client/Lykke.Service.Assets.Client/Updaters/AssetsUpdater.cs

```diff
@@ -16,7 +16,7 @@ public AssetsUpdater(IAssetsService service)
 
         public async Task<IEnumerable<Asset>> GetItemsAsync(CancellationToken token)
         {
-            return await _service.AssetGetAllAsync(false, token);
+            return await _service.AssetGetAllAsync(true, token);
```

Comment:
That's could break some clients. May be will be better to introduce new method, something like GetAllItems

Reply:
We do not call that method directly. It is like a background updater for AssetsServiceWithCache.
But in the AssetsServiceWithCache there is a method with argument includeNonTradable

Reply:
the logic before that commit is a bug